src/client/src/database.rs

```diff
@@ -130,12 +106,12 @@ impl Database {
 }
 
 #[derive(Debug)]
-pub enum ObjectResult {
-    FlightData(Vec<FlightMessage>),
-    Mutate(GrpcMutateResult),
+pub enum RpcOutput {
+    Recordbatches(RecordBatches),
+    AffectedRows(u32),
```

Comment:
Looks like mysql's `AffectedRows` uses `u64` while `FlightMessage::AffectedRows` uses `usize`

Reply:
Changed RpcOutput::AffectedRows to usize. Can't change mysql's because it's in third party crate.